13_import.sql

```diff
@@ -0,0 +1,575 @@
+  NEW.inlet_6_depth_m,
+  NEW.inlet_7_material,
+  NEW.inlet_7_clear_hight,
+  NEW.inlet_7_depth_m,
+  NEW.outlet_1_material,
+  NEW.outlet_1_clear_hight,
+  NEW.outlet_1_depth_m,
+  NEW.outlet_2_material,
+  NEW.outlet_2_clear_hight,
+  NEW.outlet_2_depth_m   
+  );
+  RETURN NEW;
+END; $BODY$
+LANGUAGE plpgsql;
+
+DROP TRIGGER IF EXISTS on_mutation_make_insert ON qgep_import.vw_manhole;
+
+CREATE TRIGGER on_mutation_make_insert
+  INSTEAD OF INSERT OR UPDATE
+  ON qgep_import.vw_manhole
+  FOR EACH ROW
+  EXECUTE PROCEDURE qgep_import.vw_manhole_insert_into_quarantine();
+
+
+-- create triggerfunctions and triggers for quarantine table 
+
+CREATE OR REPLACE FUNCTION qgep_import.manhole_quarantine_try_structure_update() RETURNS trigger AS $BODY$
+DECLARE 
+  multi_situation_geometry geometry(MultiPoint,2056);
+BEGIN
+  multi_situation_geometry = st_collect(NEW.situation_geometry)::geometry(MultiPoint,2056);
+
+  -- qgep_od.wastewater_structure
+  IF( SELECT TRUE FROM qgep_od.vw_qgep_wastewater_structure WHERE obj_id = NEW.obj_id ) THEN
+    UPDATE qgep_od.vw_qgep_wastewater_structure SET
+    identifier = NEW.identifier,
+    situation_geometry = multi_situation_geometry,
+    co_shape = NEW.co_shape,
+    co_diameter = NEW.co_diameter,
+    co_material = NEW.co_material,
+    co_positional_accuracy = NEW.co_positional_accuracy,
+    co_level = NEW.co_level,
+    _depth = NEW._depth,
+    _channel_usage_current = NEW._channel_usage_current,
+    ma_material = NEW.ma_material,
+    ma_dimension1 = NEW.ma_dimension1,
+    ma_dimension2 = NEW.ma_dimension2,
+    ws_type = NEW.ws_type,
+    ma_function = NEW.ma_function,
+    ss_function = NEW.ss_function,
+    remark = NEW.remark,
+    wn_bottom_level = NEW.wn_bottom_level
+    WHERE obj_id = NEW.obj_id;
+    RAISE NOTICE 'Updated row in qgep_od.vw_qgep_wastewater_structure';
+  ELSE
+    INSERT INTO qgep_od.vw_qgep_wastewater_structure
+    (
+    obj_id,
+    identifier,
+    situation_geometry,
+    co_shape,
+    co_diameter,
+    co_material,
+    co_positional_accuracy,
+    co_level,
+    _depth,
+    _channel_usage_current,
+    ma_material,
+    ma_dimension1,
+    ma_dimension2,
+    ws_type,
+    ma_function,
+    ss_function,
+    remark,
+    wn_bottom_level
+    )
+    VALUES
+    (
+    NEW.obj_id,
+    NEW.identifier,
+    multi_situation_geometry,
+    NEW.co_shape,
+    NEW.co_diameter,
+    NEW.co_material,
+    NEW.co_positional_accuracy,
+    NEW.co_level,
+    NEW._depth,
+    NEW._channel_usage_current,
+    NEW.ma_material,
+    NEW.ma_dimension1,
+    NEW.ma_dimension2,
+    NEW.ws_type,
+    NEW.ma_function,
+    NEW.ss_function,
+    NEW.remark,
+    NEW.wn_bottom_level
+    );
+    RAISE NOTICE 'Inserted row in qgep_od.vw_qgep_wastewater_structure';
+  END IF;
+
+  -- photo1 insert
+  IF (NEW.photo1 IS NOT NULL) THEN
+    INSERT INTO qgep_od.file
+    (
+      object,
+      identifier
+    )
+    VALUES
+    ( 
+      NEW.obj_id,
+      NEW.photo1
```

Comment:
we might some renaming on the photo filenames?

Reply:
ping @m-kuhn

Reply:
Can you precise in what way there is a renaming needed?

Reply:
Still open. What I don't understand is, that when we rename the path to the photo in the db-entry, we have to rename the existing file as well, right? Where should that be done?

Reply:
you are proposing to do some postprocessing of the filename, not changing the attribute name?

Reply:
I'm just wondering if we should make the file name a bit more unique:
* on QField side: use time stamp
* on QGEP side: use the object ID to rename the file.

But am I the only one to see this as an issue?

Reply:
I think it's a good proposal, but I don't think it's a requirement here, I think we are already over the estimated time.